app/src/main/java/com/ariel/readme/data/repo/HotWordRepository.kt

```diff
@@ -31,5 +56,31 @@ class HotWordRepository : FirebaseRepository<HotWord>(), IHotWordRepository {
         return listenOnHotWords(user.uid!!, listener)
     }
 
+    fun removeHotWord(word: String){
+        //collRef.whereEqualTo(HotWord::uid.name, FirebaseAuth.getInstance().currentUser!!.uid)
+        collRef.whereEqualTo(HotWord::uid.name, "1234")
+            .whereEqualTo(HotWord::word.name, word)
+            .get().addOnSuccessListener {
+                res -> for(doc in res.documents){
+                    try{
+                        collRef.document(doc.id).delete()
+                    }
+                    catch(e: Exception){ }
```

Comment:
how can the caller know something has failed? better return a task and let the caller register for a result